tests/check_ipc.c

```diff
@@ -429,15 +428,18 @@ run_ipc_server(void)
 		qb_ipcs_enforce_buffer_size(s1, max_size);
 	}
 	qb_ipcs_poll_handlers_set(s1, &ph);
+	qb_loop_signal_add(my_loop, QB_LOOP_HIGH, SIGTERM,
+			   NULL, exit_handler, &handle);
 
 	res = qb_ipcs_run(s1);
 	ck_assert_int_eq(res, 0);
 
 	qb_loop_run(my_loop);
+	qb_ipcs_destroy(s1);
 	qb_log(LOG_DEBUG, "loop finished - done ...");
 }
 
-static int32_t
+static pid_t
```

Comment:
Good idea. I'll put those two 'real' fixes in now. The cleanup ones rare being more elusive than I'd like